geomesa-kafka/geomesa-kafka-datastore/src/test/scala/org/locationtech/geomesa/kafka/data/KafkaDataStoreTest.scala

```diff
@@ -342,6 +342,88 @@ class KafkaDataStoreTest extends Specification with Mockito with LazyLogging {
       }
     }
 
+    "expire entries based on cql filters" >> {
+      foreach(Seq(true, false)) { cqEngine =>
+        val executor = mock[ScheduledExecutorService]
+        val ticker = new MockTicker()
+        val params = {
+          val expiry =
+            """{
+               |"name = 'smith'": "100ms",
+               |"name = 'jones'": "200ms"
+               |}""".stripMargin
```

Comment:
Should the fallback be tested?  You've got options to add to the mix: https://en.wikipedia.org/wiki/Smith_and_Jones

Reply:
what options? smith and jones are an iconic duo! ;)

Reply:
What's the obvious name of the third person for the fallback test case?!

Reply:
haha, not following you here...

Reply:
Since Smith and Jones are commonly paired together, what name do you add for the fallback test case?  

Were your Smith and Jones the agents from the Matrix?  If so, Neo or Trinity may make sense.... etc, etc...

Reply:
haha, i think the names were just a coincidence - maybe a subconscious decision